.github/workflows/ci.yml

```diff
@@ -35,26 +39,19 @@ jobs:
             environment.yml
       - name: Build new docker image
         if: env.MATCHED_FILES
-        run: docker build --no-cache . -t nfcore/pgdb:dev
```

Comment:
Do we need this here, in DSL2 I don't think we need to add the docker containers here. The nextflow will do it. Each module will download their own container or tool.

Reply:
I removed the code that built the container.